package.json

```diff
@@ -29,6 +29,7 @@
     "@react-three/fiber": "^6.1.2",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "strip-ansi": "^3.0.1",
```

Comment:
Why did it get installed in the root? When published on NPM the module will not be available, I think it will break...